doc/Programming-guide/Key-platform-features/Data-collection/Rest-log-appender/index.md

```diff
@@ -0,0 +1,252 @@
+3. Set the log appender type to <i>REST</i>.
+4. Fill in other fields as required.
+5. Click <b>Add</b>.
+
+<img src="attach/add-log-appender-in-admin-ui.png">
+
+
+It's also possible to create REST appender using [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs #TODO).
+
+# Configuration
+
+The Admin REST log appender configuration should match to
+[this](https://github.com/kaaproject/kaa/blob/master/server/appenders/rest-appender/src/main/avro/rest-appender-config.avsc) Avro schema.
+
+Parameters for defining REST log appender
+
+|parameter|description|
+|---|---|
+|connectionPoolSize|number of threads that can simultaneously perform operation with your service|
+|header|boolean value that define whether or not use a Kaa header|
+|host|a host of your custom service that will receive logs|
+|method|define a HTTP method that will be using for sending data|
+|mimeType|mime type which appender use for sending data|
+|password|password for user of service (if authentication required)|
+|path|define a path that will be able to receive logs from REST appender|
+|port|port of service|
+|ssl|boolean value that define whether or not use a SSL communication.|
+|username|name of user of your service (if authentication required)|
+|verifySslCert|boolean value that define whether or not use a verification of SSL Certificate|
+
+<br/>
+
+The following configuration schema matches to previously introduced Avro schema:
+
+```json
+{
+    "host":"10.2.2.65",
+    "port":9000,
+    "ssl":false,
+    "verifySslCert":false,
+    "username":{
+        "string":""
+    },
+    "password":{
+        "string":""
+    },
+    "connectionPoolSize":1,
+    "header":false,
+    "path":"/encrypt",
+    "method":"POST",
+    "mimeType":"JSON"
+}
+```
+
+Based on this configuration, you'd be able to perform "POST" request to 10.2.2.65:9000/encrypt url. Let's look at more definitive example.
+
+# Creating application that uses REST log appender
+
+1. Log in Admin UI like admin and create an application.
+To create a new application, do the following: <br/>
+Open the Applications window by clicking the corresponding link on the navigation panel. <br/>
+<img src="attach/add-application1.png"> <br/>
+Click <b>Add application</b> at the top of the window.
+Enter the title of your application and then click <b>Add</b>. <br/>
+<img src="attach/add-application2.png"> <br/>
+>**NOTE:**
+> If you open the Application details window of the newly created application (by clicking this application on either the Applications menu on the
+navigation panel or the Applications window), you will notice that the Application Token field has been filled in automatically. This is a unique
+auto-generated application ID.
+2. Log in as a tenant developer and create log schema in your previously created application:
+yourApp->Schemas->Log->Add schema
+<img src="attach/rest-log-appender1.png">
+upload the following configuration schema:
```

Comment:
please update screen shoots for REST Appender. They contain strange values "Robotix" and other.